### PR TITLE
remove storybook:

### PR DESCRIPTION
**Pull request by Monkeychip**
As a team we decided to remove storybook. We didn't use it often enough and it was becoming a burden to update it's dependencies. 

[Here](https://github.com/hashicorp/vault/pull/6496/files) was the original PR that added storybook. 

I also updated browserstack. Not entirely sure why, but after removing the storybook dependencies from the package.json file, browserstack started to fail. I updated it and it fixed it. I wish I knew the answer to these things, but sometimes the web is too thick.

### CODE REVIEW

**Review by hellobontempo — COMMENTED** [review]
LGTM! But I'll hold off reviewing and leave it to someone with more experience changing dependencies. 

Let me know if you want to pair on tackling that vercel failure

**Review by hashishaw — APPROVED** [review]
Thank you!

**Review by zofskeez — APPROVED** [review]
This looks good to me thanks for tackling!

**Pull request merged**
